tests/gen/long_no_extended.toml

```diff
@@ -0,0 +1,2 @@
+bin.name = "eza"
+args = "tests/test_dir --long --extended"
```

Comment:
test is not revelent name and args dont match and the execution does not change anything please add extended attribute generation

Reply:
You are totally right, I'm sorry for botching this. Obviously it should be --no-extended, but I'm struggling with getting a proper test in, because the xattr, which is the basis for the extended attributes have no proper tests anywhere. Adding them would be a separate PR wouldn't it?